packages/review/src/components/review-container.tsx

```diff
@@ -338,15 +338,11 @@ function ReviewContainer({
 
         {shortened ? (
           <>
+            <Text bold size="huge" color="gray" alpha={1} inline>
+              {t(['ribyu', '리뷰'])}
+            </Text>
             {(totalReviewsCount || 0) > 0 ? (
-              <Trans
-                i18nKey={[
-                  'ribyu-totalreviewscount',
-                  '<0>리뷰</0><1> {{totalReviewsCount}}</1>',
-                ]}
-                ns="common-web"
-              >
-                <Text bold size="huge" color="gray" alpha={1} inline />
+              <Trans i18nKey={['totalreviewscount']} ns="common-web">
```

Comment:
```suggestion
              <Text bold size="huge" color="blue" alpha={1} inline>
                {formatNumber(totalReviewsCount)}
              </Text>
```

여기는 단순히 숫자만 보여주는 케이스로 변경돼서 국제화를 적용하지 않아도 될듯 합니다!
`Trans` 컴포넌트를 사용하지 않아도 될 듯 하고, `common-web.ts` 파일에 있는 `totalreviewscount` key또한 제거해도 될 듯 합니다 🙇

Reply:
헉 생각해보니 그렇군요! 0801af272c372a27a65e0b8dfee2c475f545c094 에서 수정했습니다. 알려주셔서 감사합니다🙋‍♀️